semgrep_output_v1.atd

```diff
@@ -554,71 +568,16 @@ type error_span <ocaml attr="deriving show"> = {
     ?context_end: position nullable option;
   }
 
-(*****************************************************************************)
-(* CLI call trace *)
-(*****************************************************************************)
-
-type cli_match_call_trace
-  <ocaml attr="deriving show">
-  <python decorator="dataclass(frozen=True, order=True)"> = [
-  | CliLoc of (location * string)
-  | CliCall of ((location * string) * cli_match_intermediate_var list * cli_match_call_trace)
-] <ocaml repr="classic">
-
-(* from rule_lang.py in the Span class
- * "The path to the pattern in the yaml rule
- *  and an adjusted start/end within just the pattern
- *  Used to report playground parse errors in the simpler editor"
- *)
-(* EXPERIMENTAL *)
-type cli_match_dataflow_trace
-  <ocaml attr="deriving show">
-  <python decorator="dataclass(frozen=True)"> = {
-  ?taint_source: cli_match_call_trace option;
-  (* Intermediate variables which are involved in the dataflow. For taint, this
-   * explains how the taint flows from the source to the sink. *)
-  ?intermediate_vars: cli_match_intermediate_var list option;
-  (* For now, omitting the taint sink, since it's redundant data except for
-   * certain cases that only appear with DeepSemgrep. We can add it in later
-   * without breaking consumers. *)
-  ?taint_sink: cli_match_call_trace option;
-}
-
-(* EXPERIMENTAL *)
-type cli_match_taint_source
```

Comment:
@IagoAbal @nmote I've move around the section up, but I also removed this cli_match_taint_source which appeared dead.

Reply:
Sounds great as long as everything still works